### PR TITLE
10.0 Add hr_recruitment_candidate_multi_applicant

### PR DESCRIPTION
**Pull request by leemannd**
Add the possibility to see multiple applications made by one partner

### CODE REVIEW

**Comment by leemannd** [comment]
@gurneyalex Ready for review

File: hr_recruitment_candidate_multi_applicant/README.rst
Comment (gurneyalex) [inline hr_recruitment_candidate_multi_applicant/README.rst]: s/candiate/candidate/

File: hr_recruitment_candidate_multi_applicant/README.rst
Comment (gurneyalex) [inline hr_recruitment_candidate_multi_applicant/README.rst:9]: explain that "same candidate is detected by same mail or same phone"

File: hr_recruitment_candidate_multi_applicant/models/hr_applicant.py
Comment (gurneyalex) [inline hr_recruitment_candidate_multi_applicant/models/hr_applicant.py:32]: probably the domain building should be extracted in a helper method to make it easier to extend the module.

File: hr_recruitment_candidate_multi_applicant/models/hr_applicant.py
Comment (gurneyalex) [inline hr_recruitment_candidate_multi_applicant/models/hr_applicant.py]: use  `active_test = False`in the search and remove the active = True or active = False in the domain.

**Review by gurneyalex — CHANGES_REQUESTED** [review]
a few changes suggested

**Comment by leemannd** [comment]
@gurneyalex Updated. Thanks for the feedback. To be squashed before merge.

**Review by gurneyalex — APPROVED** [review]

**Comment by leemannd** [comment]
Squashed

File: hr_recruitment_candidate_multi_applicant/models/hr_applicant.py
Comment (leemannd) [reply hr_recruitment_candidate_multi_applicant/models/hr_applicant.py:32]: @feketemihai I'm putting me a reminder to do it this week or early the next one

**Review by feketemihai — APPROVED** [review]
The extraction of domain was good, but not a blocker.

File: hr_recruitment_candidate_multi_applicant/__init__.py
Comment (hieulucky111) [inline hr_recruitment_candidate_multi_applicant/__init__.py:2]: Missing `-*- coding: utf-8 -*-` : https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms

File: hr_recruitment_candidate_multi_applicant/models/__init__.py
Comment (hieulucky111) [inline hr_recruitment_candidate_multi_applicant/models/__init__.py:2]: Missing -*- coding: utf-8 -*- : https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms

File: hr_recruitment_candidate_multi_applicant/models/hr_applicant.py
Comment (hieulucky111) [inline hr_recruitment_candidate_multi_applicant/models/hr_applicant.py]: Alphabetically ordered: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

File: hr_recruitment_candidate_multi_applicant/models/hr_applicant.py
Comment (hieulucky111) [inline hr_recruitment_candidate_multi_applicant/models/hr_applicant.py]: variable name should be `same_apps`

File: hr_recruitment_candidate_multi_applicant/tests/__init__.py
Comment (hieulucky111) [inline hr_recruitment_candidate_multi_applicant/tests/__init__.py:3]: Missing -- coding: utf-8 -- : https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms

File: hr_recruitment_candidate_multi_applicant/tests/test_multi_applicant.py
Comment (hieulucky111) [inline hr_recruitment_candidate_multi_applicant/tests/test_multi_applicant.py:31]: Same with `test_compute_same_email`?

**Review by hieulucky111 — APPROVED** [review]
Some minor adjustments, LGTM over all

File: hr_recruitment_candidate_multi_applicant/tests/test_multi_applicant.py
Comment (leemannd) [reply hr_recruitment_candidate_multi_applicant/tests/test_multi_applicant.py:31]: @hieulucky111 I don't undersand your point. The email is tested in => https://github.com/OCA/hr/pull/373/files/fa410c9efb6192e0ce51a251679d122e6d61e033#diff-59fe9f77b440f08b6354e3e4b21b3720R26

File: hr_recruitment_candidate_multi_applicant/views/applicant.xml
Comment (nikul-serpentcs) [inline hr_recruitment_candidate_multi_applicant/views/applicant.xml]: here use `hasclass`

**Review by nikul-serpentcs — COMMENTED** [review]
Small Change

File: hr_recruitment_candidate_multi_applicant/views/applicant.xml
Comment (nikul-serpentcs) [inline hr_recruitment_candidate_multi_applicant/views/applicant.xml:9]: Remove `icon` and add `<i class="fa-users" />`

**Comment by leemannd** [comment]
Closing my work that is 1 year old

**Comment by leemannd** [comment]
Closing my work that is 1 year old

File: hr_recruitment_candidate_multi_applicant/models/hr_applicant.py
Comment (leemannd) [reply hr_recruitment_candidate_multi_applicant/models/hr_applicant.py:32]: I'm wondering, as it's old. Can we get it merge and improve it if needed?

**Comment by leemannd** [comment]
@rafaelbn As you are at a CS, can you take a look at it?
Maybe we want to get it merged?

If there are changes to be done, please amend my PR or feel free to close it.

**Review by rafaelbn — APPROVED** [review]
I'm not PSC of HR but this PR should be merged

**Comment by rafaelbn** [comment]
@gurneyalex could merge this

**Comment by nikul-serpentcs** [comment]
@gurneyalex @rafaelbn squashing commits are remaining :smile_cat:

**Pull request merged**
